src/uris/deepLinks/deepLinkService.ts

```diff
@@ -119,35 +161,61 @@ export class DeepLinkService implements Disposable {
 		return undefined;
 	}
 
-	private async processDeepLink(): Promise<void> {
+	private async showOpenRepoPrompt(): Promise<OpenWorkspaceLocation | undefined> {
+		const result = await window.showInformationMessage(
+			`No matching repository found. Please choose an option to open the repository.`,
+			{ modal: true },
+			{ title: 'Open in Current Window', action: OpenWorkspaceLocation.CurrentWindow },
```

Comment:
`'Open'` rather than `'Open in Current Window'`, since it matches how we and VS Code have similar prompts.

Reply:
Done in commit: [Update verbiage, open repo flow](https://github.com/gitkraken/vscode-gitlens/pull/2503/commits/2154cca1a37764d8f4416e8b37a0ff75a990a2d0)